setup.py

```diff
@@ -1,12 +1,27 @@
+import re
+import subprocess
 from distutils.core import setup
 
 from setuptools import find_packages
 
 import server
 
+
+def get_version() -> str:
+    output = subprocess.run(
+        ["git", "describe", "--abbrev=0", "--tags"],
+        capture_output=True
+    ).stdout.decode().strip()
+
+    m = re.match(r"v?(\d+\.\d+\.\d+)", output)
+    if not m:
+        return "dev"
+    return m.groups()[0]
+
+
 setup(
     name='Forged Alliance Forever Server',
-    version=server.__version__,
+    version=get_version(),
     packages=['server'] + find_packages(),
     url='http://www.faforever.com',
     license=server.__license__,
```

Comment:
do we need that file at all?

Reply:
You mean `setup.py`? It is used to by the dockerfile to install the server. Not sure what the advantage is of doing it this way, but it doesn't hurt.